### ISSUE TITLE
Wrong commit highlighted in bold

### ISSUE THREAD

**Issue by Reubend** [ForkIssues/Tracker#657]
I recently updated to Fork 1.0.80 and noticed a new issue with the commit list. Sometimes, the list doesn't update which commit is bolded, even as the current branch has updated. Here's a screenshot of what I mean:
 
<img width="429" alt="Capture" src="https://user-images.githubusercontent.com/13208038/59645422-8359dc80-9126-11e9-84cc-bcc1f75ee54c.png">

I started at the commit in bold, merged in the newer one, and the branch indicator correctly updated. But the newer commit didn't get highlighted in bold. For reference, I'm *not* in a detached head state, so the commit on top should be in bold. 

I'm fairly sure this bug was introduced in the last few versions, because it only started occuring when I updated today.

P.S. Unrelated to this bug, I think Fork is fantastic software! Keep up the great work :)

**Comment by DanPristupov** [comment]
Hi. Thank you for the bug report.

> I started at the commit in bold, merged in the newer one, and the branch indicator correctly updated. 

Can you reproduce that?
Did you make the usual merge operation?
Is your repository located on some remote file system?

**Comment by Reubend** [comment]
I use a local volume with a case-sensitive filesystem. Your comment got me thinking, and I did some testing. I can reproduce it consistently using the following steps.

1. Create a volume with a case-sensitive filesystem (Unix style).
2. Use Fork to initialize a new repo within that volume.
3. Make a file in the repo, and use Fork to stage it and commit it.
4. Fork will show the initial commit, but won't highlight it in bold:

![Capture](https://user-images.githubusercontent.com/13208038/59704891-083e0800-91b2-11e9-86da-2202aed32d30.png)


I would expect that because it's the current (and only) commit, it would be highlighted in bold. But it gets shown in light grey instead of bold.

For debugging purposes, I'm using macOS High Sierra (10.13.6) with Fork 1.0.80

**Comment by DanPristupov** [comment]
> Create a volume with a case-sensitive filesystem (Unix style).

How can I create such a volume? Is that a logical disk? Could you point me to some docs please?
How does the absolute path for the repository look in your case? Is it something like `/Volumes/volume1/repositories/Testing/`?

**Comment by Reubend** [comment]
You can use the GUI tool [Disk Utility](https://support.apple.com/en-gb/guide/disk-utility/welcome/mac) that comes with macOS to do it. Use [this guide](https://support.apple.com/en-gb/guide/disk-utility/dskutl11888/mac) to create the volume with these settings:

- **Format**: Mac OS Extended (Case-sensitive, Journaled)
- **Partitions**: Single Partition - Apple Partition Map
- **Image Format**: Sparse Bundle Disk Image

and then mount and open it.


However, if you prefer, you can do all this with the command line instead:
```
hdiutil create -verbose -type SPARSEBUNDLE -fs 'Case-sensitive Journaled HFS+' -size 5g -volname Unix ~/Unix.sparsebundle
hdiutil attach ~/Unix.sparsebundle
open ~/Unix.sparsebundle
 
cd /Volumes/Unix
```
Those commands will make a 5 GB volume for you with the correct settings, and then `cd` into it.

**Comment by klapec** [comment]
I can reproduce this as well and just like for @Reubend it happens for me on a case-sensitive APFS volume.
Here's the screencast:
![Wrong commit highlighted in bold](https://thumbs.gfycat.com/IckyVictoriousFreshwatereel-size_restricted.gif)

**Comment by DanPristupov** [comment]
Thank you guys. I've reproduced the problem and fixing it now.

**Comment by DanPristupov** [comment]
I released the hotfix here: https://forkapp.ams3.cdn.digitaloceanspaces.com/mac/Fork-1.0.80.1.dmg

I won't force all Fork users to update to it, so please download and install it manually.

Could you try it and give me some feedback please?

**Comment by klapec** [comment]
I can confirm that fix introduced in 1.0.80.1 does indeed work. Thanks for the quick response! :)

**Comment by Reubend** [comment]
I can confirm that your hotfix resolves this issue, and restores the correct behavior. I'm closing this issue now. Thanks for the fast fix!
